hashes/polynomial_hash.py

```diff
@@ -0,0 +1,25 @@
+"""
+Calculate the hash of a string using a polynomial rolling hash function.
```

Comment:
If this is a polynomial rolling hash, then why not name the file and function `polynomial_rolling_hash`?